Packages/com.unity.inputsystem/Documentation~/ActionBindings.md

```diff
@@ -0,0 +1,223 @@
+# Action Bindings
+
+----mention/explain disambiguation on this page
+
+An `InputBinding` represents a connection between an action and one or more controls identified by a [control path](Controls.md#control-paths). An action can have arbitrary many bindings pointed at it and the same control may be referenced by multiple bindings.
```

Comment:
Again, why is this page called Action Binding, when the actual class name is `InputBinding`?